packages/react/src/Form/InputText/Text.tsx

```diff
@@ -18,6 +20,7 @@ const Text = forwardRef<HTMLInputElement, Props>(
     return (
       <input
         {...otherProps}
+        id={inputId}
```

Comment:
```suggestion
        id={id ?? inputUseId}
```
Why declare the `inputId` constant when it has only one use ?